saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -1168,6 +1174,12 @@ def handle_api_response(
         raw_response=response.message,
         action_required_data=response.message.get("action"),
         psp_reference=response.message.get("pspReference", ""),
+        legacy_adyen_plugin_payment_method=response.message.get("paymentMethod", "")
+        .strip()
+        .lower(),
+        legacy_adyen_plugin_result_code=response.message.get("resultCode", "")
+        .strip()
+        .lower(),
```

Comment:
I'm not sure if this is zero-downtime compatible? 

is there a way to gracefully drop these fields when columns don't exist yet?

Reply:
Yes, read will make a fallback, but what about write. Is adding new column instant? What if django tries to write these 2 fields before migration is done?